kyuubi-server/src/main/scala/org/apache/kyuubi/engine/KubernetesApplicationOperation.scala

```diff
@@ -61,6 +61,9 @@ class KubernetesApplicationOperation extends ApplicationOperation with Logging {
         val operation = findDriverPodByTag(tag)
         val podList = operation.list().getItems
         if (podList.size() != 0) {
+          if (podList.get(0).getStatus.getPhase.equalsIgnoreCase("Terminated")) {
```

Comment:
shall we use `KubernetesApplicationOperation. toApplicationState` ?

Reply:
Done